src/gtfs/mod.rs

```diff
@@ -562,3 +562,16 @@ pub fn write<P: AsRef<Path>>(model: Model, path: P) -> Result<()> {
 
     Ok(())
 }
+
+/// Exports a `Model` to [GTFS](https://gtfs.org/reference/static) files
+/// in the given ZIP archive.
+/// see [NTFS to GTFS conversion](https://github.com/CanalTP/transit_model/blob/master/src/documentation/ntfs2gtfs.md)
+pub fn write_to_zip<P: AsRef<std::path::Path>>(model: Model, path: P) -> Result<()> {
+    let path = path.as_ref();
+    info!("Writing NTFS to ZIP File {:?}", path);
```

Comment:
if it's ntfs to zip, shoudln't this be in src/ntfs/mod.rs?

Reply:
Oops, some copy/paste that I didn't change... Done in def0c255.